app/src/main/java/com/example/plantsapp/domain/repository/TasksRepository.kt

```diff
@@ -6,5 +6,6 @@ import java.util.Date
 
 interface TasksRepository {
 
+    suspend fun addTasks(tasks: List<Task>)
```

Comment:
It will be more cool if your repository will know task for WHAT plant are you adding
Not just some random tasks